google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/SecurityMarkSnippets.java

```diff
@@ -44,7 +45,11 @@ private SecurityMarkSnippets() {}
   // [START securitycenter_add_security_marks]
   static SecurityMarks addToAsset(String assetName) {
     try (SecurityCenterClient client = SecurityCenterClient.create()) {
-      // String assetName = "organizations/123123342/assets/12312321";
+      // assetName: must be in one of the following formats:
```

Comment:
"Specify the value of assetName in one of the following formats:"?

Reply:
Done. Thanks